examples/NonReg/SPI_loop/SPI_loop.ino

```diff
@@ -0,0 +1,81 @@
+/*
+  SPI_loopback
+
+  Test the loopback transfer on SPI serial link.
+  A test string is sent 9 times and compared to the Rx frame.
+  when the MISO pin is physically connected to the MOSI pin of the SPI bus
+  Default test the SPI_A link. To test the SPI_B transfer, just define SPI_B
+  to change the pin
+
+  This example code is in the public domain.
+
+*/
+#include "SPI.h"
+
+#if defined(SPI_B)
```

Comment:
SPI_B is not a standard definition. Where does it come from ?

Reply:
SPI_B_xx like SPI_A_xx as defined by the schematics